perllib/FixMyStreet/Cobrand/Camden.pm

```diff
@@ -34,4 +34,42 @@ sub privacy_policy_url {
 
 sub admin_user_domain { 'camden.gov.uk' }
 
+sub lookup_site_code_config {
+    my ($self, $property) = @_;
+
+    # uncoverable subroutine
+    # uncoverable statement
+    {
+        buffer => 1000, # metres
+        url => "https://tilma.mysociety.org/mapserver/camden",
+        srsname => "urn:ogc:def:crs:EPSG::27700",
+        typename => "Streets",
+        property => $property,
+        accept_feature => sub {
+            # Sometimes the nearest feature has a NULL streetref1 property
```

Comment:
These `streetref1` references should be `NSG_REF`, based on L67.

Reply:
Also there aren't any null entries, so not sure the whole function is needed :)